releasenotes/notes/add-sql-obfuscator-options-and-update-cgo-bindings-078a7fe8bf91076e.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Adds support to provide options to the SQL obfuscator to change the behavior.
+enhancements:
+  - |
+    The quantize_sql_tables obfuscator option is now configured at the integration level via the instance config.
```

Comment:
All trace-agent related release notes need to be prefixed with `APM: `

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8556/commits/d7e46cde49253718503b756a70f0254356f7d4e8